sweep/log.go

```diff
@@ -27,19 +27,3 @@ func DisableLog() {
 func UseLogger(logger btclog.Logger) {
 	log = logger
 }
-
```

Comment:
If anything that uses `spew` is ever logged, then we'll need this as it lets us defer those operations until log time. Otherwise, we'll always call into reflect, even if the logging level won't be hit (on debug and the log is trace for example).

Reply:
Commit dropped